source/test/F0.Minesweeper.Components.Tests/Logic/Cell/CellVisualisationManagerTests.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using F0.Minesweeper.Components.Abstractions.Enums;
+using F0.Minesweeper.Components.Logic.Cell;
+using FluentAssertions;
+using Xunit;
+
+namespace F0.Minesweeper.Components.Tests.Logic.Cell
+{
+	public class CellVisualisationManagerTests
+	{
+		[Fact]
+		public void GetVisualisation_UncoveredWithoutAdjacentMineCount_Throws()
+		{
+			// Arrange
+			CellVisualisationManager instanceUnderTest = new();
+
+			// Act && Assert
+			Func<CellVisualisation> methodUnderTest = () => instanceUnderTest.GetVisualisation(CellStatusType.Uncovered);
+			methodUnderTest.Should().Throw<ArgumentNullException>();
```

Comment:
Well, .NET does not derive from `ArgumentNullException` any further, but my OCD would still prefer `ThrowExactly` here 😟